example/lib/snapshot_scene.dart

```diff
@@ -0,0 +1,91 @@
+import 'dart:math' as math;
+import 'package:arkit_plugin/arkit_plugin.dart';
+import 'package:flutter/material.dart';
+import 'package:vector_math/vector_math_64.dart' as vector;
+
+class SnapshotScenePage extends StatefulWidget {
```

Comment:
Awesome!! Could you please add an item with this page to the list in `main.dart`? In that case, plugin users will see the proposed functionality just by playing around with the sample.